internal/network/command_network_link_service.go

```diff
@@ -0,0 +1,121 @@
+package network
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	networkingv1 "github.com/confluentinc/ccloud-sdk-go-v2/networking/v1"
+
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+type networkLinkServiceHumanOut struct {
+	Id                   string `human:"ID"`
+	Name                 string `human:"Name"`
+	NetworkId            string `human:"Network ID"`
```

Comment:
In many of our commands, we drop the "ID" here since it's implied from the output.
```suggestion
	Network              string `human:"Network"`
```

Reply:
(I see you've already done this below with "Environment")

Reply:
Updated!